packages/strapi-design-system/vite.config.js

```diff
@@ -0,0 +1,36 @@
+import { resolve } from 'path';
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import glob from 'tiny-glob';
+
+export default glob('./src/**/!(*.spec|*.e2e).{js,svg}').then(async (paths) => {
+  console.log(paths, resolve(__dirname, './src/index.js'));
```

Comment:
Might be a leftover.

Reply:
> I found only one thing that I'm not sure of: dist/index.cjs contains only require statements and not the whole bundle. Is that intentional? Would e.g. webpack resolve these dependencies even if we do not transpile node_modules and consume cjs files? I think in the ESM case it does, but I'm not sure about cjs.

Yes, I've re-linked it with the `admin` app and it's does still build. The `exports` part of the `package.json` helps bundlers know which file is for `require` and which for `import` from there it's just a matter of following the trail.

Realistically it would be simpler to just bundle everything into one file because the direct import is unnecessary now (we could action removing `@strapi/design-system/Select` in the admin app when we release this version if you agree) because components aren't duplicated, but if we did that here then it would break plugins 😄

Reply:
> The exports part of the package.json helps bundlers know which file is for require and which for import from there it's just a matter of following the trail.

💡 Thanks! I knew it did in the case of esm (`module`), but it's good to know it does also for cjs (`main`).

> Realistically it would be simpler to just bundle everything into one file because the direct import is unnecessary now (we could action removing @strapi/design-system/Select in the admin app when we release this version if you agree) because components aren't duplicated but if we did that here then it would break plugins 😄

Yes, I agree and think we should settle for one direction in the future. It would also simplify the imports in e.g. strapi core a lot, if we could use one import statement.

Reply:
Cool, I am happy to action this when I update the admin app with the new DS 👍🏼